src/pages/dmt5-trading-signals/_how-to.tsx

```diff
@@ -1,22 +1,32 @@
 import React from 'react'
 import styled from 'styled-components'
 import SignalSteps from './_signal-steps'
-import { localize, Localize } from 'components/localization'
+import { Localize } from 'components/localization'
 import { SectionContainer, Container } from 'components/containers'
 import { Header } from 'components/elements'
 import { LinkButton } from 'components/form'
 import device from 'themes/device'
+import { TString } from 'types/generics'
 
 type HowToProps = {
     active_tab: 'signal-subscriber' | 'signal-provider'
 }
 
-const content = {
+type TContent = {
     subscriber: {
-        header: <Localize translate_text="How to subscribe to an MT5 signal" />,
+        header: TString
+    }
+    provider: {
+        header: TString
+    }
+}
+
+const content: TContent = {
```

Comment:
Let's refactor it a little bit 
```suggestion
type TContent = {
  subscriber: TString
  provider: TString
}

const content: TContent = {
     subscriber: '_t_How to subscribe to an MT5 signal_t_',
    provider: '_t_How to register as a signals provider_t_',
}
const header_text: TString = signal_subscriber ? content.subscriber : content.provider
```